index.html

```diff
@@ -5,4 +5,39 @@
     <link rel="stylesheet" type="text/css" href="css/reset.css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
+  <body>
+    <div>
+      
+    </div>
+    <main>
+    
+    <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add Brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirrinf frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+    <p>Ice cream: In a small pot over medium heat, combiine mik, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled. </p>
+    <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture,a nd the nuts. Continue the process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+    <div class="container"></div>
+    <form action=" /action_page.php" method="get">
```

Comment:
I'm not sure where this example of a form came from, but we've never used forms with actions or methods before!